mycroft/skills/skill_loader.py

```diff
@@ -26,6 +26,7 @@
 from mycroft.util.log import LOG
 
 from mycroft.skills.settings import SettingsMetaUploader
+from mycroft.skills.mycroft_skill.mycroft_skill import MycroftSkill
```

Comment:
I hate that also, and since i made all imports explicit its even worse in ovos-core....

Again, we cant refactor if we care about backwards compat, we need to live with the ugly for now...